backend/src/osm_tests/README.md

```diff
@@ -0,0 +1,20 @@
+# OSM unit tests
+
+Importing existing modal filters and turn restrictions from OSM can be very complicated. This directory contains minimal synthetic OSM inputs to test different situations.
+
+## Creating a unit test
+
+1.  Open the iD editor and zoom in on an empty patch of ocean, [like here](https://www.openstreetmap.org/edit#map=17/55.704282/-0.110917)
+2.  Create the simplest example for some test case
+3.  Press **Save** in the top-right corner
+4.  Go to the bottom and **Download osmChange file**. Do **not** actually upload this edit!
+5.  Turn the .osc into osm.xml and add it to this directory: `osmium cat ~/Downloads/changes.osc -o my_test_case.osm.xml`
+6.  Add a test case in `mod.rs`, following examples there
+
+## Viewing the OSM unit test inputs
+
+To understand a unit test input:
+
+-  Just manually read it -- many are small enough to understand
+-  Load in <https://dabreegster.github.io/geojson-viewer/> -- but note relations don't show up
```

Comment:
😍 I didn't know you had an osm.xml viewer, nice! Can you share the source for it?

Reply:
It's a quick hacked-together GeoJSON viewing tool that winds up being faster for me to use than opening QGIS and figuring out how to symbolize by different properties. Has more property-level detail than geojson.io, and more free (and faster to load) than Felt. Adding OSM support was just gluing into a library that does the hard work:
https://github.com/dabreegster/geojson-viewer/commit/6fd5301a926bae56336a20c24a59ec0400b2ec30

Reply:
Dustin, you're writing javascript apps that aren't backed by rust. I'm growing concerned! 🙃 😉

Reply:
I've done my penance with widgetry! Time to enjoy all the wonderful and only slightly confusing web technologies!